### ISSUE TITLE
NextCloud login not supported

### ISSUE THREAD

**Issue by hellomiakoda** [MM2-0/Kvaesitso#959]
It takes me to the log in, I put in my username and password, and then it says it doesn't support WebAuthn

**Comment by exstntlmsnthrp** [comment]
You might need to share more details and a debug log. That's not exactly enough information for someone to try to duplicate the problem.

I'm logged in to Nextcloud without any problems.

**Comment by s-slash** [comment]
Same issue here.
I got 2FA enable on the nextcloud server (hub 8 with https) and log using an app password lead to "does not support webauth" and using id, password and 2FA leads to a loop on "grant acces" button that never stop. I already log into other app using webauth or classic logging on the same nextcloud server (and version).

**Comment by exstntlmsnthrp** [comment]
Yes, that's identical to what I have setup. I think the login loop is a Nextcloud issue. I've encountered this with a different app. 

However, using an app password, I'm logged in on kvaesitso. 

![kvaesitsonc.png](https://github.com/user-attachments/assets/1f1b8bc5-5da9-4665-b375-b22c4b052a3e)
